chains/fractal/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+blockExplorers:
+  - apiUrl: https://evm.fractal.org/api-docs
+    family: blockscout
+    name: Fractal Confluence Explorer
+    url: https://evm.fractal.org
+blocks:
+  confirmations: 1
+  estimateBlockTime: 2
+  reorgPeriod: 0
+chainId: 12788
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Fractal Confluence
+domainId: 12788
```

Comment:
had to change ID to 2152 I think, for fractal confluence https://docs.fra.tech/network-settings/network-settings